dev/AppLifecycle/EncodedLaunchExecuteCommand.cpp

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "EncodedLaunchExecuteCommand.h"
+#include "ActivationRegistrationManager.h"
+
+using namespace winrt;
+using namespace winrt::Windows::Foundation;
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    IFACEMETHODIMP EncodedLaunchExecuteCommand::Execute() noexcept try
+    {
+        THROW_HR_IF_NULL(E_INVALIDARG, m_selection);
+
+        com_ptr<IShellItem> item;
+        THROW_IF_FAILED(m_selection->GetItemAt(0, item.put()));
```

Comment:
>THROW [](start = 8, length = 5)

Use RETURN_IF_FAILED inside this function (rather than THROW_IF_FAILED) since it returns HRESULT? RETURN_... is equivalent but more efficient